packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.ts

```diff
@@ -6,43 +6,126 @@ import {
-  canPauseRecording: boolean = null;
-  canResumeRecording: boolean = null;
-  canStopRecording: boolean = null;
-  canRaiseHand: boolean = null;
-  canLowerAllHands: boolean = null;
-  canLowerSomeoneElsesHand: boolean = null;
-
-  get() {
-    return {
-      canInviteNewParticipants: this.canInviteNewParticipants,
-      canAdmitParticipant: this.canAdmitParticipant,
-      canLock: this.canLock,
-      canUnlock: this.canUnlock,
-      canAssignHost: this.canAssignHost,
-      canStartRecording: this.canStartRecording,
-      canPauseRecording: this.canPauseRecording,
-      canResumeRecording: this.canResumeRecording,
-      canStopRecording: this.canStopRecording,
-      canRaiseHand: this.canRaiseHand,
-      canLowerAllHands: this.canLowerAllHands,
-      canLowerSomeoneElsesHand: this.canLowerSomeoneElsesHand,
-    };
-  }
-
-  set(actions) {
+  canInviteNewParticipants: boolean | null = null;
+
+  canAdmitParticipant: boolean | null = null;
+
+  canLock: boolean | null = null;
+
+  canUnlock: boolean | null = null;
+
+  canAssignHost: boolean | null = null;
+
+  canStartRecording: boolean | null = null;
+
+  canPauseRecording: boolean | null = null;
+
+  canResumeRecording: boolean | null = null;
+
+  canStopRecording: boolean | null = null;
+
+  canRaiseHand: boolean | null = null;
+
+  canLowerAllHands: boolean | null = null;
+
+  canLowerSomeoneElsesHand: boolean | null = null;
+
+  canEnableClosedCaption: boolean | null = null;
+
+  canStartTranscribing: boolean | null = null;
+
+  canStopTranscribing: boolean | null = null;
+
+  isClosedCaptionActive: boolean | null = null;
+
+  isWebexAssistantActive: boolean | null = null;
+
+  canViewCaptionPanel : boolean | null = null;
+
+  isRealTimeTranslationEnabled: boolean | null = null;
+
+  canSelectSpokenLanguages : boolean | null = null;
+
+  /**
+   * Returns all meeting action options
+   * @returns {Object}
+   */
+  get = () => ({
+    canInviteNewParticipants: this.canInviteNewParticipants,
+    canAdmitParticipant: this.canAdmitParticipant,
+    canLock: this.canLock,
+    canUnlock: this.canUnlock,
+    canAssignHost: this.canAssignHost,
+    canStartRecording: this.canStartRecording,
+    canPauseRecording: this.canPauseRecording,
+    canResumeRecording: this.canResumeRecording,
+    canStopRecording: this.canStopRecording,
+    canRaiseHand: this.canRaiseHand,
+    canLowerAllHands: this.canLowerAllHands,
+    canLowerSomeoneElsesHand: this.canLowerSomeoneElsesHand,
+    canEnableClosedCaption: this.canEnableClosedCaption,
+    canStartTranscribing: this.canStartTranscribing,
+    canStopTranscribing: this.canStopTranscribing,
+    isClosedCaptionActive: this.isClosedCaptionActive,
+    isWebexAssistantActive: this.isWebexAssistantActive,
+    canViewCaptionPanel: this.canViewCaptionPanel,
+    isRealTimeTranslationEnabled: this.isRealTimeTranslationEnabled,
+    canSelectSpokenLanguages: this.canSelectSpokenLanguages
+  });
+
+  set = (actions:any) => {
```

Comment:
please do not use `any` especially when the type is known. In this case it is the whole `IInMeetingActions` or just some part of it
```suggestion
  set = (actions: Partial<IInMeetingActions>) => {
```

Reply:
refactored